### PR TITLE
Remove incorrect quotes for POSTFIX_smtpd_recipient_restrictions

### PR DESCRIPTION
**Pull request by wader**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ environment:
   - POSTFIX_smtpd_sasl_auth_enable=yes
   - POSTFIX_broken_sasl_auth_clients=yes
   - POSTFIX_smtpd_sasl_security_options=noanonymous
-  - POSTFIX_smtpd_recipient_restrictions="permit_sasl_authenticated, permit_mynetworks, reject_unauth_destination"
+  - POSTFIX_smtpd_recipient_restrictions=permit_sasl_authenticated,permit_mynetworks,reject_unauth_destination"
```
Comment (rubencm) [inline README.md:74]: Remove the end quote too

Comment (rubencm) [inline README.md:74]:
```suggestion
  - POSTFIX_smtpd_recipient_restrictions=permit_sasl_authenticated,permit_mynetworks,reject_unauth_destination
```

Comment (wader) [reply README.md]: Doh, fixed

**Review by rubencm — APPROVED** [review]
It works now. And thank you for this image

**Comment by wader** [comment]
@rubencm Great and thanks for reporting!

**Pull request merged**